misk-hibernate/src/test/kotlin/misk/hibernate/TransacterTest.kt

```diff
@@ -78,6 +80,7 @@ class TransacterTest {
   }
 
   @Test
+  @Disabled("Uniqueness constraints aren't reliably enforced on Vitess")
```

Comment:
Yep. We have to create a new test that is reliable under Vitess.

Reply:
Yeah not sure how to do that... :-) Uniqueness constraints isn't something that scales out well. The best alternative is cross shard leases. Maybe we should have something built-in that supports that pattern.

Reply:
Yeah I think the goal of the test wasn't uniqueness, just something that'd fail in SQL. Perhaps we could force an error by duplicating a primary key.

Reply:
Gotcha. I'll figure something out...